charts/kubewarden-controller/tests/webhooks_existing_certifcates_test.yaml

```diff
@@ -0,0 +1,71 @@
+suite: existing certificates are not overwritten
+templates:
+  - webhooks.yaml
+release:
+  namespace: "kubewarden"
+kubernetesProvider:
+  scheme:
```

Comment:
Maybe add an inline comment explaining we're simulating these resources to be already defined inside of the cluster while `helm` is run